File: tests/test64.py
def plugin_test (x, d):
	print("Python got "+str(x)+" from C")
	if d == 0 :
		anirudh.return_val_i(x)
	else:
		anirudh.return_val_d(x)

anirudh.plugin_test = plugin_test

File: tests/test64.rb
def Anirudh.plugin_test(x, d)
	#print ("Ruby got "+x.to_s+" from C\n");
	if d==0
		Anirudh.return_val_i(x)
	else
		Anirudh.return_val_d(x)
	end
end